docs/litmus-admin-rbac.yaml

```diff
@@ -14,9 +14,12 @@ metadata:
   labels:
     name: litmus-admin
 rules:
-- apiGroups: ["","apps","batch","extensions","litmuschaos.io","openebs.io","storage.k8s.io"]
-  resources: ["chaosengines","chaosexperiments","chaosresults","cstorpools","cstorvolumereplicas","configmaps","secrets","pods","pods/exec","pods/log","pods/eviction","jobs","replicasets","deployments","daemonsets","statefulsets","persistentvolumeclaims","persistentvolumes","storageclasses","services","events"]
+- apiGroups: ["","apps","batch","extensions","litmuschaos.io"]
+  resources: ["pods","pods/exec","pods/eviction","jobs","daemonsets","events","chaosresults","chaosengines"]
   verbs: ["create","delete","get","list","patch","update", "deletecollection"]
+- apiGroups: ["","apps","litmuschaos.io"]
```

Comment:
I used below for my pod experiment ```- apiGroups: ["","litmuschaos.io","batch","apps"]
  resources: ["pods","deployments","events","jobs","configmaps","chaosengines","chaosexperiments","chaosresults","pods/log"]
  verbs: ["create","list","get","patch","update","delete"]
- apiGroups: [""]
  resources: ["nodes"]
  verbs : ["get","list"]```

Reply:
Yes, this is fine. 

We may need statefulsets and ds as well.

`"configmaps","secrets","services","chaosexperiments","pods/log","replicasets","deployments","statefulsets",services`

^^ These don't need the create and delete permissions